### PR TITLE
automl: add natural language sentiment analysis ga samples

### PR DESCRIPTION
**Pull request by nnegrey**

### CODE REVIEW

File: automl/cloud-client/language_sentiment_analysis_create_model_test.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import os
+
+from google.cloud import automl
+import pytest
+
+import language_sentiment_analysis_create_model
+
+PROJECT_ID = os.environ["GCLOUD_PROJECT"]
+DATASET_ID = "TST3960250460385409610"
```
Comment (kurtisvg) [inline automl/cloud-client/language_sentiment_analysis_create_model_test.py]: If this is a resource we expect the user to have created before hand, it should be in an environment variable (and trigger an error if it isn't set when the tests are run)

File: automl/cloud-client/language_sentiment_analysis_predict_test.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import os
+
+from google.cloud import automl
+import pytest
+
+import language_sentiment_analysis_predict
+
+PROJECT_ID = os.environ["GCLOUD_PROJECT"]
+MODEL_ID = "TST8532792392862639819"
```
Comment (kurtisvg) [inline automl/cloud-client/language_sentiment_analysis_predict_test.py]: Same as above

File: .kokoro/tests/run_tests.sh [hunk 3]
```diff
@@ -39,6 +39,7 @@ SECRETS_PASSWORD=$(cat "${KOKORO_GFILE_DIR}/secrets-password.txt")
 source ./testing/test-env.sh
 export GOOGLE_APPLICATION_CREDENTIALS=$(pwd)/testing/service-account.json
 export GOOGLE_CLIENT_SECRETS=$(pwd)/testing/client-secrets.json
+source "${KOKORO_GFILE_DIR}/automl_secrets.txt"
```
Comment (nnegrey) [inline .kokoro/tests/run_tests.sh]: @kurtisvg, I'm getting a key error for the environment variables. Is there something I missed on setting this one up?

Error: `KeyError: 'AUTOML_PROJECT_ID'`

Comment (kurtisvg) [reply .kokoro/tests/run_tests.sh]: I think it's fixed. Ping me again if still an issue.

**Comment by nnegrey** [comment]
@kurtisvg, moved everything over to the automl project for this PR, let me know if there is anything else on this one.

**Pull request merged**